### PR TITLE
Put the names that the validator uses for the boilerplate into the spec

### PR DESCRIPTION
**Pull request by powdercloud**
The validator prints errors such as these:
**The tag 'head > style : boilerplate - old variant' is deprecated - use 'head > style : boilerplate' instead. Learn more.**
**The tag 'noscript > style : boilerplate - old variant' is deprecated - use 'noscript > style : boilerplate' instead. Learn more.**
It can't just say "AMP Boilerplate" because it's two tags, so the names have to be slightly lengthy here. I'd like to put these names into the documentation so that a user who clicks the "Learn more." links will more easily find the boilerplate that the message told them is missing, e.g. in case they're searching for it with CTRL+F. It should also help with search engines.

### CODE REVIEW

**Comment by honeybadgerdontcare** [comment]
LGTM, but will wait for label flipping till docs owners chime in.

**Comment by powdercloud** [comment]
Actually one more sec I'll add some to it.

File: spec/amp-boilerplate.md [hunk 1]
```diff
@@ -14,7 +14,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# AMP Boilerplate Code
+# AMP Boilerplate Code ('head > style : boilerplate' and 'noscript > style : boilerplate')
```
Comment (dvoytenko) [inline spec/amp-boilerplate.md]: If you following the CSS/DOM selectors style, then it'd be:
```
head > style[boilerplate]
```

Comment (powdercloud) [inline spec/amp-boilerplate.md]: Done.

Comment (powdercloud) [inline spec/amp-boilerplate.md]: Ah one more - it should be amp-boilerplate actually. Will update it.

**Comment by powdercloud** [comment]
@dvoytenko @honeybadgerdontcare @Gregable Ready for review. I also sent an internal cl for changing this in the validator so it uses the style that Dima suggested.

**Comment by Gregable** [comment]
lg

**Comment by honeybadgerdontcare** [comment]
lgtm^2

**Pull request merged**